### ISSUE TITLE
Bug: Tooltip the tip disappears when being moved over with a pointer 

### ISSUE THREAD

**Issue by kellyhutchins** [Esri/calcite-components#938]
### Summary
The configurable apps team is undergoing an accessibility review from Level Access and they pointed out that the tooltip disappears when being moved over by a pointer.  
It looks like this behavior violates [WCAG 2.1 1.4.13 guidelines](https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus).  

I found this [article](https://sarahmhigley.com/writing/tooltips-in-wcag-21/) which has some details on this in the 'WCAG 2.1 dismissable, hoverable and persistant' section. The issue Level Access pointed out is in regards to the 3rd bullet in that section: 

_Allow a mouse user to move their mouse over the tooltip content without dismissing the tooltip -- ideally in a manner that does not require laser focus and precision mouse control._

### Actual Behavior
The tooltip disappears when being moved over the pointer 

### Expected Behavior
Ensure that the pointer can be moved to the new content without the content disappearing. Hover label should remain active when moved over with pointer.

FYI @driskull

**Comment by driskull** [comment]
@kellyhutchins can you verify this one?

**Comment by kellyhutchins** [comment]
@driskull  are the changes out on the calcite sample page?

**Comment by driskull** [comment]
I think so
